src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -152,10 +153,11 @@ protected void updateItem(final TableRow item, final boolean empty) {
                      setText(null);
                   } else {
                      final String notes = item.getNotes();
-                     final Label label = new Label(notes.isEmpty() ? EMPTY_NOTE : notes);
-                     label.setUnderline(item.isUnderlined());
-                     label.setTooltip(new Tooltip(notes));
-                     this.setGraphic(label);
+                     final Text text = new Text(notes.isEmpty() ? EMPTY_NOTE : notes);
+                     this.setText(text.getText());
```

Comment:
we do not need the "Text" class when in the end you just pass a String to the setText method. A normal String is sufficient ;)